### ISSUE TITLE
file extensions are case-sensitive

### ISSUE THREAD

**Issue by gcaptn** [sprout2000/tauview#2]
Leafview refuses to open images unless they're all lowercase (e.g. `.JPG` won't open, but `.jpg` works). I've only tested this on my Mac.

**Comment by sprout2000** [comment]
@gcaptn 
Thanks for letting me know.
Although I could not reproduce it on my Mac (APFS + not case-sensitive), I tried to fix it.
Could you try v0.4.2 and let me know how it goes?

https://github.com/sprout2000/leafview-tauri/releases/tag/v0.4.2

**Comment by gcaptn** [comment]
Yup! It works fine. A mistake on my part because I didn't realize I was on a really old build (from back when this was tauview) 🤦 Thank you so much for the quick response!
